controllers/depresolver/depresolver_config.go

```diff
@@ -55,6 +57,32 @@ const (
 	MetricsAddressKey              = "METRICS_ADDRESS"
 )
 
+// Deprecated environment variables keys
+const (
+	// Deprecated: Please use EDGE_DNS_SERVERS instead.
+	EdgeDNSServerKey = "EDGE_DNS_SERVER"
+
+	// Deprecated: Please use EDGE_DNS_SERVERS instead.
+	EdgeDNSServerPortKey = "EDGE_DNS_SERVER_PORT"
+)
+
+type oldVar = string
+type newVar struct {
+	Name string
+	Msg  string
+}
+
+var deprecated = map[oldVar]newVar{
+	EdgeDNSServerKey: newVar{
+		Name: EdgeDNSServersKey,
+		Msg:  "Pass the hostname or IP address as comma-separated list",
+	},
+	EdgeDNSServerPortKey: newVar{
+		Name: EdgeDNSServersKey,
+		Msg:  "Port is an optional in the comma-separated list of dns edge servers, in following form: dns1:53,dns2",
+	},
+}
```

Comment:
I didn't want the function to do it over and over again, but I guess that would be my cognitive baggage from my previous job on graalvm compiler (premature optimisation, yada yada..) I've learnt that package scoped vars are evil so I will move that to that function as u suggested because in our case, the function is called only once and even if it's called bazzilion times, I assume the golang compiler can do inlining and escape analysis these days. 

As for the suggested new form: I need to also keep track of the "old form" -> "new form" so that user is informed about what is the remedy for the deprecation (check `depresolver_config.go:L264`).